src/modules/Comments.js

```diff
@@ -0,0 +1,65 @@
+const apiUrlForCoin = (coin) => `https://api.coingecko.com/api/v3/coins/${coin}?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false`;
+
+const readableNumbers = (int) => int.toString()
+  .split('').reverse()
+  .map((el, i) => ((i % 3 === 0 && i > 0) ? `${el},` : el))
+  .reverse()
+  .join('');
+
+const digitRemover = (int) => (int > 0 ? Math.floor(int) : Math.ceil(int));
+const baseCurrency = 'usd';
+const symbolMap = { usd: '$' };
+
+export default (coin) => {
+  const popupFrame = document.querySelector('#popup-frame');
+  const popupInner = popupFrame.querySelector('#popup-inner');
+  popupFrame.classList.remove('d-none');
+  fetch(apiUrlForCoin(coin))
+    .then((result) => result.json())
+    .then((parsed) => {
+      popupInner.innerHTML = '';
+      console.log(parsed);
+      const {
+        name,
+        symbol,
+        image: { large: imageSrc },
+        market_data: { current_price: { [baseCurrency]: currentPrice } },
+        market_data: { market_cap: { [baseCurrency]: marketCap } },
+        market_data: { total_volume: { [baseCurrency]: totalVolume } },
+        market_data: {
+          price_change_24h: priceChange24H,
+          price_change_percentage_1y: priceChange1Y,
+        },
+      } = parsed;
+      popupInner.innerHTML += `
+            <div class="position-relative p-3 m-3 border border-2 border-dark">
+                <span class="position-absolute cursor-pointer" id="popup-close">X</span>
+                <img src="${imageSrc}">            
+                <h2 class="mt-3">${name}</h2>
```

Comment:
i think using small function is a good way to implement a big functionality, code with small functions is easier to read and understand and also when implementing tests you can't detect were  a problem is coming from without small functions.

Reply:
I didnt understand what you mean by small function.